freedomofpress/securedrop.org
# Update to python 3.12.1,  django 4.2, wagtail 5.2

## Description

Fixes #1077.

Changes proposed in this pull request:
- Updates to django 4.2 and make necessary code changes
- Updates to Wagtail 4.2 and make necessary code changes
- Updates to Python 3.12 and updates other python packages to support Python 3.12

## Type of change



## Testing

How should the reviewer test this PR?
It might be worth checking the changelogs to see if I missed anything. Other than that, try building the container locally with the new dependencies, check all the pages and admin section and tests work without throwing any errors or deprecation warning.
